app/app.js

```diff
@@ -50,14 +50,15 @@ const ARTICLES = require('./content').ARTICLES;
 
 // Info.
 if (console.log) {
-  console.log('Scenic started. Publication: ' + PUBLICATION_ID);
+  console.log('Scenic started. Publication: ' + getPublicationId());
 }
 
 
 /**
  * List all Articles.
  */
-app.get('/', (req, res) => {
+app.get(['/','/config/:configId'], (req, res) => {
```

Comment:
LG, but the same question: why do we need "/config/" in the path? Could we just do `/:configId/...` URLs?

Reply:
done